### ISSUE TITLE
Add library

### ISSUE THREAD

**Issue by lind** [bric3/excalidraw-jetbrains-plugin#16]
Nice to have Excalidraw as a IntelliJ pluging 😄 

Is it possible to add library in the plugin like in excalidraw.com?

Plugin:
<img width="458" alt="Skjermbilde 2021-08-19 kl  21 51 44" src="https://user-images.githubusercontent.com/75105/130135045-c065beea-59d6-4b53-8574-9153f36fb2d1.png">

excalidraw.com
<img width="458" alt="Skjermbilde 2021-08-19 kl  21 53 24" src="https://user-images.githubusercontent.com/75105/130135337-61d49553-ca7f-4bba-958e-b180ac579600.png">

Plugin version used 0.3.1

**Comment by bric3** [comment]
@lind Yes I would like to support that at some point. But I haven't looked at how to do it at this time. I mostly focused on making the plugin working offline.

(And iron UI things first.)

Any contribution would be welcome.
